packages/common/src/services/submissionService.ts

```diff
@@ -0,0 +1,81 @@
+import { isEmpty } from 'lodash-es';
+import { Dependencies } from '../config/config.js';
+import dictionaryUtils from '../utils/dictionaryUtils.js';
+import submissionUtils, { emptySubmission, parseToResultSubmission } from '../utils/submissionUtils.js';
+import {
+	BatchError,
+	CreateActiveSubmission,
+	CreateSubmissionResult,
+	DictionaryData,
+	SubmissionEntity,
+} from '../utils/types.js';
+
+const service = (dependencies: Dependencies) => {
+	const LOG_MODULE = 'SUBMISSION_SERVICE';
+	const { logger } = dependencies;
+	return {
+		/**
+		 * Creates and validates the Entities Schemas of the Active Submission and stores it in the database
+		 * @param {SubmissionEntity[]} submissionsEntities An array of Entities within the Submission
+		 * @param {number} categoryId Category ID of the Submission
+		 * @returns The Active Submission created or Updated
+		 */
+		uploadSubmission: async (
+			submissionsEntities: SubmissionEntity[],
+			categoryId: number,
+		): Promise<CreateSubmissionResult> => {
+			logger.info(
+				LOG_MODULE,
+				`Processing '${submissionsEntities.length}' Submission entities on category '${categoryId}'`,
+			);
+			const { createOrUpdateActiveSubmission, mappingEntities } = submissionUtils(dependencies);
+			const { getCurrentDictionary } = dictionaryUtils(dependencies);
+
+			let resultSubmission: CreateActiveSubmission = emptySubmission();
+			let batchErrors: BatchError[] = [];
+
+			if (submissionsEntities.length > 0) {
+				const currentDictionary = await getCurrentDictionary(categoryId);
+				// TODO: map this dictionary field as a 'DictionaryData' type on Drizzle models
+				// Dictionary data stores as a JSONB type in database
+				const dictionaryData: DictionaryData[] = currentDictionary.dictionary as DictionaryData[];
```

Comment:
so far not running any DictionarySchemas to validate. Will require use corresponding `SchemaDefinition` type to run validations